appyx-interactions/common/src/commonMain/kotlin/com/bumble/appyx/transitionmodel/cards/interpolator/CardsProps.kt

```diff
@@ -53,9 +53,13 @@ class CardsProps<NavTarget : Any>(
         ),
         val rotationZ: RotationZ = RotationZ(value = 0f),
         val zIndex: ZIndex = ZIndex(value = 0f),
-        override val isVisible: Boolean = false
+        private val width: Float,
     ) : Interpolatable<Props>, HasModifier, Animatable<Props>, BaseProps {
 
+        override val isVisible: Boolean
+            get() = scale.value >= 0.0f && positionalOffsetX.value.x > (-voteCardPositionMultiplier * width).dp &&
```

Comment:
Also need to take `zIndex` into account, as the bottom card will be covered by the top card. Wouldn't it be easier to rely (fully or partially) on an enum value in this class instead of trying to incorporate all UI properties? It's a simplification that in most cases would still be right.